site/content/docs/5.2/utilities/z-index.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: docs
+title: z-index
```

Comment:
```suggestion
title: Z-index
```

For consistency with https://deploy-preview-37317--twbs-bootstrap.netlify.app/docs/5.2/layout/z-index/.